corehq/apps/export/models/new.py

```diff
@@ -222,7 +222,7 @@ def _transform(self, value, doc, transform_dates, domain):
         if transform_dates:
             value = couch_to_excel_datetime(value, doc)
         if self.item.transform:
-            value = TRANSFORM_FUNCTIONS[self.item.transform](value, doc)
+            value = TRANSFORM_FUNCTIONS[self.item.transform](value, doc, domain)
```

Comment:
Thanks @benrudolph 
The signatures being called already have the argument added. It was missing when called.

i think @calellowitz would look into once he is back since the underlying changes were reverted.